### DISCUSSION TITLE
`MetronomeDisplay` strange when update thread is slow

### DISCUSSION THREAD

**Discussion by cdwcgt** [ppy/osu#19381] [Q&A]
Phenomenon:
Looks like it's stuck, shaking erratically, but still on the right beat 

It's very likely that this has something to do with the clock.
But this is rare because it is difficult to have slow update threads in modern computers.
The slower the update thread, the more serious it is
this may not be a problem, but I still leave it as a question though

https://user-images.githubusercontent.com/34775378/180780522-08d67590-6fd3-465c-806a-09367fc2cd93.mp4

possibly related:
- https://github.com/ppy/osu/issues/18365

**Comment by frenzibyte** [comment]
This is likely happening as a result of disabling `AllowMistimedEventFiring` in metronomes. That said, I'm not sure a slow update thread will have metronomes playing in time? Would have to double-check myself to be sure, but it'll likely play the metronome sound a bit late than usual.
